### ISSUE TITLE
Questions about training settings and dataset for live2diff

### ISSUE THREAD

**Issue by somuchtome** [open-mmlab/Live2Diff#7]
Hi, Thank you for the amazing work! I am currently working on a similar project, and your work is a important baseline for us.

I have a few questions about the training process:
1. which scheduler do you choose during training? is it LCM or DDIM?
2. Could you share more details about the training datasets? For example the dataset size.

Thank you in advance for your help!

**Comment by LeoXing1996** [comment]
Hey @somuchtome, thanks for your interest in our project.

1. We use the naive DDPM schedule during training (same as AnimateDiff)
2. We use WebVid10M dataset, you can download the annotation from this [url](https://huggingface.co/datasets/TempoFunk/webvid-10M).
